File: ProjectEuler/Problem_5/5-Smallest_multiple.cs
// Find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20

// Mathematical shortcut / multiplying primes < 20
void Main()
{
    int x = 19 * 17 * 13 * 11 * 7 * 5 * 3 * 2; //9699690
    bool found = false;
    while (!found)
    {
        for (int j = 1; j < 21; j++)
        {
            if (x % j > 0) break;
            else if (j == 20 && x % j == 0)
            {
                Console.WriteLine("Smallest multiple number is: {0}", x);
                found = true;
            }
        }
        x += 9699690;
    }
}
// Smallest multiple number is: 232792560

File: ProjectEuler/Problem_5/5-Smallest_multiple.js
// Find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20

let x = 19*17*13*11*7*5*3*2 //9699690

let found = false
while (!found) {
    for (let i = 1; i < 21; i++) {
        if (x % i) break
        else if (i == 20 && x % i == 0) {
            console.log(`Smallest multiple number is: ${x}`) //232792560
            found = true;
        }
    }
    x += 9699690
}
